paper-api/src/main/java/org/bukkit/entity/Creaking.java

```diff
@@ -1,8 +1,45 @@
 package org.bukkit.entity;
 
+import org.bukkit.Location;
+import org.jspecify.annotations.NullMarked;
+
 /**
  * Represents a Creaking.
  */
+@NullMarked
 public interface Creaking extends Monster {
 
+    /**
+     * Gets the home location for this Creaking (aka where can be a {@link org.bukkit.block.CreakingHeart}).
+     *
+     * @return the location of the home.
+     */
+    Location getHome();
+
+    /**
+     * Sets the home location for this Creaking.
+     *
+     * @param location the location of the home.
+     */
```

Comment:
Needs a `@throws` for the different world part, if that's actually how it must go.